setup.py

```diff
@@ -31,7 +31,7 @@ def get_snakemake_pkg(extras=""):
 extras_require = {
     "docs": ["Sphinx>=1.4.4", "sphinx-rtd-theme>=0.1.9",],
     "tests": tests_require,
-    "kubernetes": ["kubernetes>=11.0.0,<12.0.0",],
+    "kubernetes": ["kubernetes>=21.0.0,<22.0.0",],
```

Comment:
I see the `kubernetes` library now has version 22.6 released on PyPI. Is it interesting to release upper boundary constraint?

Reply:
We can. In this case, the k8s client will fully cover the 1.22 version. As I understood, we are targeting it.

Reply:
We can. Should I release it completely or just increase the upper boundary to `23.0.0`? I would prefer to put an upper boundary to `<23.0.0`

Reply:
I put version  `>=22,<23`, for now, because it fixes the mock issue. Should we remove the upper boundary?

Reply:
Yes, please keep the upper boundary; It'll be safer.